src/d2_docker/config/dhis2-core-entrypoint.sh

```diff
@@ -49,6 +52,18 @@ if [ "$(id -u)" = "0" ]; then
         rm -v $WARFILE # just to save space
     fi
 
+    if [ -f $GLOWROOT_ZIP ] && [ ! -d $GLOWROOT_DIR ] ; then
+        unzip -q $GLOWROOT_ZIP -d /opt/ || [ $? -le 1 ]
```

Comment:
that's a bit cryptic. I gather code=1 in unzip is non-terminal, and we can continue. I'd refactor it so the intent is more clear, something like:

```
status=0
unzip -q "$GLOWROOT_ZIP" -d /opt/ || status=$?
if [ $status -gt 1 ]; then
  exit $status
fi
```